### ISSUE TITLE
LocalList::read with default $pageSize value causes divide by zero error 

### ISSUE THREAD

**Issue by rtek** [twilio/twilio-php#388]
```
//failed invocation
$instances = $client->availablePhoneNumbers->getContext('US')->local->read([
  //etc
], 1);

//LocalList
public function read($options = array(), $limit = null, $pageSize = Values::NONE) {
    return iterator_to_array($this->stream($options, $limit, $pageSize), false);
}

public function stream($options = array(), $limit = null, $pageSize = null) {
    $limits = $this->version->readLimits($limit, $pageSize);
    //...
}

//Version
public function readLimits($limit = null, $pageSize = null) {
    $pageLimit = Values::NONE;

    if ($limit) {
        if (is_null($pageSize)) {
             $pageSize = min($limit, self::MAX_PAGE_SIZE);
        }
        $pageLimit = (int)(ceil($limit / (float)$pageSize));
    }
//....
}
```

The default $pageSize value for LocalList::read() should be null so that Version will properly use MAX_PAGE_SIZE.

**Comment by ihumanable** [comment]
This looks like a bug affecting all `read()`s, will triage for fix today

**Comment by efossier** [comment]
Fixed in 5.5.0
